client/client.go

```diff
@@ -172,11 +172,7 @@ func (r *NotaryRepository) Initialize(rootKeyID string) error {
 	err = r.tufRepo.InitRoot(false)
 	if err != nil {
 		logrus.Debug("Error on InitRoot: ", err.Error())
-		switch err.(type) {
-		case signed.ErrInsufficientSignatures, trustmanager.ErrPasswordInvalid:
-		default:
-			return err
-		}
+		return err
```

Comment:
It's possible I had `InitRoot` attempting to sign the root at some point in the past. I don't remember it doing that though, and it wouldn't be OK to continue if the signing had failed.